rosa_tutorials/rosa-mobb-configure-cluster-pull-images-aws-ecr-tutorial.adoc

```diff
@@ -0,0 +1,175 @@
+:_content-type: ASSEMBLY
+[id=“rosa-mobb-configure-cluster-pull-images-aws-ecr-tutorial”]
+= Tutorial: Configuring a ROSA cluster to pull images from AWS Elastic Container Registry
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: rosa-mobb-<topic>-tutorial
+
+toc::[]
+
+//Mobb content metadata
+//Brought into ROSA product docs 2023-09-20
+//---
+//date: '2022-04-26'
+//title: Configuring a ROSA cluster to pull images from AWS Elastic Container Registry (ECR)
+//weight: 1
+//tags: ["AWS", "ROSA"]
+//authors:
+//  - Kevin Collins
+//  - Byron Miller
+//---
+== Prerequisites
```

Comment:
What users should be used for below oc command run? cluster-admin? Then it also should be a prerequisite